hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/featurestore/FeaturestoreController.java

```diff
@@ -220,36 +228,50 @@ public FeaturestoreDTO convertFeaturestoretoDTO(Featurestore featurestore) {
     featurestoreDTO.setHdfsStorePath(hdfsPath);
     Long inodeId = featurestoreFacade.getFeaturestoreInodeId(featurestore.getHiveDbId());
     featurestoreDTO.setInodeId(inodeId);
+    String hiveEndpoint = settings.getHiveServerHostName(false);
+    featurestoreDTO.setHiveEndpoint(hiveEndpoint);
+    featurestoreDTO.setOfflineFeaturestoreName(hiveDbName);
+    if(settings.isOnlineFeaturestore() &&
+      onlineFeaturestoreController.checkIfDatabaseExists(featurestore.getProject().getName())) {
+      String hostname = settings.getHopsworksIp();
```

Comment:
There is an option in hopsworks-chef to specify the jdbc url of the online feature store. Shouldn't the enpoint return that?

Reply:
Done